code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -139,6 +139,12 @@
 	results = list("lizardmutationtoxin" = 1)
 	required_reagents = list("unstablemutationtoxin" = 1, "radium" = 1) //mutant
 
+/datum/chemical_reaction/ethari_mutation_toxin
+	name = "Ethari Mutation Toxin"
+	id = "etharimutationtoxin"
+	results = list("etharimutationtoxin" = 1)
+	required_reagents = list("unstablemutationtoxin" = 1, "welding_fuel" = 1)
```

Comment:
A different recipe, perhaps? Most of the stuff in similar ones isn't hard to obtain at all.